src/GeoPing.FrontEnd/src/componentContainers/checkListComponentContainer.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import IinitialStateType from '../DTO/types/stateTypes/initialStateType';
+import { CheckListGeoPointComponent } from '../components/checkListGeoPointComponent';
+import { ListPointsComponent } from '../components/listPointsComponent';
+import ICheckListComponentContainerProps from '../componentProps/checkListComponentContainerProps';
+import { checkGEOPosition, editingPermission } from '../actions/checkListAction';
+import {
+  addNewPoint,
+  cancelAddNewPoint,
+  cancelEditingGEOPoint,
+  changeDataGEOPoint, deleteMarker,
+  editGEOPoint, markerInstalled,
+  permissionToAddMarker, putStatusMarker,
+  selectedMarker
+} from '../actions/googleMapAction';
+import { CheckListLinkComponent } from '../components/checkListLinkComponent';
+import { addNotification, deleteNotification } from '../actions/notificationsAction';
+
+class CheckListComponentContainer extends React.Component<ICheckListComponentContainerProps, any> {
```

Comment:
Again, too much props passed, review possibility of separating concerns, or at least gather them into one object in constructor or *render* method and than passing it like <Componen {...componentProps} />

Notifications for sure should be top level component, or be managed via some kind of redux action call.